pkg/parser/common.go

```diff
@@ -32,7 +33,11 @@ import (
 func Parse(p *parser.Parser, sql, charset, collation string) (stmt []ast.StmtNode, err error) {
 	stmts, warnings, err := p.Parse(sql, charset, collation)
 	if err != nil {
-		log.L().Error("parse statement", zap.String("sql", sql), log.ShortError(err))
+		if utils.IsBuildInSkipDDL(sql) {
+			log.L().Warn("parse statement", zap.String("sql", sql), log.ShortError(err))
```

Comment:
seems useless, although this ddl should be skipped, will also return error

Reply:
Do you mean that the outer function will print this error?

Reply:
yes

Reply:
Revised. The error log will be printed when function `CheckIsDDL` of relay module is deployed. Now we will check the skipped ddl before parse in `CheckIsDDL`.